app/Http/Controllers/App/InboxController.php

```
@@ -51,40 +53,23 @@ public function index($mail = null): Response
-                'mailable_type' => Contact::class,
-                'mailable_id' => $request->validated('contact_id'),
-            ]);
+        return redirect()->route('app.inbox.index');
+    }
 
-            if ($request->validated('project_id')) {
-                $dropboxMail->links()->create([
-                    'mailable_type' => Project::class,
-                    'mailable_id' => $request->validated('project_id'),
-                ]);
+    public function processAttachmentsAsReciept(MailImportRequest $request, $mail)
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)